src/sentry/organizations/absolute_url.py

```diff
@@ -97,3 +97,20 @@ def organization_absolute_url(
     if fragment:
         parts.append(fragment)
     return "".join(parts)
+
+
+def api_absolute_url(
+    *, slug: str, path: str, query: str | None = None, fragment: str | None = None
+) -> str:
+    path = f"/api/0/organizations/{slug}{path}"
```

Comment:
Will paths always be `/` prefixed? I wonder if we need to handle this case more thoroughly.